x-pack/solutions/security/plugins/security_solution_serverless/server/task_manager/usage_reporting_task.ts

```diff
@@ -36,6 +37,9 @@ export class SecurityUsageReportingTask {
   private readonly logger: Logger;
   private readonly config: ServerlessSecurityConfig;
   private readonly usageReportingService: UsageReportingService;
+  private readonly backfillConfig: BackfillConfig = {
```

Comment:
fyi @CohenIdo. This is disabled by default so I don't foresee any change in behavior but letting you know since this is a shared component.